docs/governance/CONTRIBUTING.md

```diff
@@ -84,7 +84,7 @@ In some cases, other changes may conflict with your PR. If this happens, you wil
 
 ## Development Environment Setup
 
-You can easily setup a developer environment by following the instructions [here](https://github.com/ovn-org/ovn-kubernetes/blob/master/docs/kind.md).
+You can easily setup a developer environment by following the instructions [here](../installation/launching-ovn-kubernetes-on-kind.md).
```

Comment:
https://ovn-kubernetes.io/#why-choose-ovn-kubernetes-in-kubernetes-ecosystem here as well we need the update